identity-credential/README.md

```diff
@@ -1 +1,46 @@
-Types and traits for working with Verifiable Credentials/Presentations.
+# Iota Identity: Credentials 
+
```

Comment:
Why do we have two `README.md` files for the same crate? This seems repetitive and confusing.

Reply:
There should only be one. Must have forgotten to delete the previous one I guess.